docs/user_guide/method/method.rst

```diff
@@ -0,0 +1,2384 @@
+    - An *optional* integer representing the number of points, :math:`N`, along the spectroscopic
+      dimension. For example, ``4096``. The default value is ``1024``.
+
+  * - spectral_width
+    - ``float``
+    - An *optional* float representing the width, :math:`\Delta x`, of the spectroscopic dimension
+      in Hz. For example, ``10e3`` for 10 kHz. The default value is ``25000`` Hz.
+
+  * - reference_offset
+    - ``float``
+    - An *optional* float representing the reference offset, :math:`x_0`, of the spectroscopic
+      dimension in Hz. For example, ``-8000`` Hz. The default value is ``0``.
+
+  * - origin_offset
+    - ``float``
+    - An optional float representing the origin offset, or Larmor frequency, along the
+      spectroscopic dimension in units of Hz. The default value is ``None`` and the origin offset
+      is set to the Larmor frequency of isotope from the :attr:`~mrsimulator.Method.channels`
+      attribute of the method containing the spectral dimension.
+
+  * - events
+    - ``List``
+    - An *optional* list of :ref:`event_api` objects used to emulate an experiment.
+      The default value is a list with a single **SpectralEvent** with a symmetry_query of
+      P=[-1]
+
+
+.. cssclass:: table-bordered table-striped centered
+.. _table_spectral_event:
+.. list-table:: The attributes of a SpectralEvent object
+  :widths: 20 15 65
+  :header-rows: 1
+
+  * - Attribute Name
+    - Type
+    - Description
+
+  * - magnetic_flux_density
+    - ``float``
+    - An *optional* float describing the macroscopic magnetic flux density of the applied
+      external magnetic field in tesla. For example, ``18.8`` tesla. The default value is
+      ``None`` and takes the global magnetic flux density defined by
+      :attr:`~mrsimulator.Method.magnetic_flux_density`.
+
+  * - rotor_angle
+    - ``float``
+    - An *optional* float describing the angle between the sample rotation axis and the external
+      magnetic field in radians. The default is ``None`` and takes the global rotor angle defined
+      by :attr:`~mrsimulator.Method.rotor_angle`.
+
+  * - rotor_frequency
+    - ``float``
+    - An *optional* float describing the sample rotation frequency in Hz. For example, ``2000`` Hz.
+      The default value is ``None`` and takes the global rotor frequency defined by
+      :attr:`~mrsimulator.Method.rotor_frequency`.
+
+  * - freq_contrib
+    - ``List``
+    - An *optional* list of :ref:`freq_contrib_api` ((object?)) selecting which frequency
+      contributions to include when calculating the spectrum. For example,
+      ``["Shielding1_0", "Shielding1_2"]``. By default, the list is all frequency enumerations and
+      all frequency contributions are calculated.
+
+  * - transition_query
+    - ``dict`` or :ref:`transition_api`
+    - An *optional* ``dict`` or :ref:`transition_api` selecting transitions active
+      during the event. Only these selected transitions will contribute to the net frequency.
```

Comment:
The type of transition query is a list of TransitionQuery objects. The default is a list of one TransitionQuery with P=0 for ch1. Would it be simple to have it an empty list by default?